index.html

```diff
@@ -947,620 +974,1301 @@
-                  Impeachment Delayed
+              <div class="col-lg-3 Us-small px-0">
+                <div class="container pr-0">
+                  <div class="row border-bottom pb-2">
+                    <div class="col-lg-4 px-0 pt-1">
+                      <img
+                        src="./assets/trump-kwame.JPG"
+                        alt="trump lowers sentence for kwame"
+                        class="img-fluid Us-small-img"
+                      />
+                    </div>
+                    <div class="col-lg-8 Us-small-text pr-0 pt-1">
+                      <strong
+                        >'Tiger King' Zoo Owners Ordered To Hand Over Big Cats
+                        to Government</strong
+                      >
+                    </div>
+                  </div>
                 </div>
-                <div class="text">
-                  "Unity starts with accountability," Warren said Sunday during
-                  an interview with CNN's Abby Phillip.
+                <div class="container pr-0 pt-1">
+                  <div class="row border-bottom pb-2">
+                    <div class="col-lg-4 px-0 pt-1">
+                      <img
+                        src="./assets/trump-kwame.JPG"
+                        alt="trump lowers sentence for kwame"
+                        class="img-fluid Us-small-img"
+                      />
+                    </div>
+                    <div class="col-lg-8 Us-small-text pr-0 pt-1">
+                      <strong
+                        >'Tiger King' Zoo Owners Ordered To Hand Over Big Cats
+                        to Government</strong
+                      >
+                    </div>
+                  </div>
+                </div>
+                <div class="container pr-0 pt-1">
+                  <div class="row pb-2">
+                    <div class="col-lg-4 px-0 pt-1">
+                      <img
+                        src="./assets/trump-kwame.JPG"
+                        alt="trump lowers sentence for kwame"
+                        class="img-fluid Us-small-img"
+                      />
+                    </div>
+                    <div class="col-lg-8 Us-small-text pr-0 pt-1">
+                      <strong
+                        >'Tiger King' Zoo Owners Ordered To Hand Over Big Cats
+                        to Government</strong
+                      >
+                    </div>
+                  </div>
                 </div>
               </div>
             </div>
-            <div class="nstorty">
-              <img src=./assets/cindy-mccain-doug-ducey-funeral.jpg alt="biden"
-              class="images-float-left">
-              <div class="heading_text">
-                <div class="bheading">
-                  Warren Calls for Unity 'Against Insurrection' as Trump
-                  Impeachment Delayed
+          </div>
+        </div>
+        <div class="row">

```

Comment:
- [ ] Let's use the section tag for this section to be semantic.
Check more how the section tag [works here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section).